ui/design/src/components/ReadOnlyEditor/image-overlay.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { ModalContainer } from '../SignInModal/fullscreen-modal-container';
```

Comment:
Codacy has a fix for the issue: [Replace `{·ModalContainer·}·from·'../SignInModal/fullscreen-modal-container` with `styled·from·'styled-components`](https://app.codacy.com/gh/AKASHAorg/akasha-world-framework/pullRequest?prid=8098754)
```suggestion
import styled from 'styled-components';
```